src/server/controllers/auth.js

```diff
@@ -29,11 +30,18 @@ router.post('/login', (req, res) => {
       });
     } 
     else {
-      req.session.authenticated = true;
-      req.session.user = mapColumns(records[0].fields);
-      res.json({
-        user: req.session.user
-      });
+      var u = user.newUser(mapColumns(records[0].fields, 'auth'));
```

Comment:
- [ ] Use const/let over var, favor utilizing const first.
- [ ] Please use meaningful names

This should probably be something like:

```js
const user = userUtil.newUser(...)
```

Reply:
The name change was necessary given the poorly-named library. It can be reinstated immediately after that change :D